doc/api-client.md

```diff
@@ -39,13 +39,29 @@ That's why D3 services provide registration of new client via HTTP POST request.
     
     **Code:** `200` <br />
     
-    **Content:** `myname@d3`
+    **Content:** 
+    
+        {
+            "clientId": "myname22@d3"
+        }
               
 * **Example error response:**
     
     **Code:** `500` <br />
     
-    **Content:** `Response has been failed. java.lang.Exception: Tx 9fd23b3372dd772ca17b400d3abecf2160d1f6d8e0afc7d7b2f5c4de956051fc failed. CreateAccount`
+    **Content:** 
+    
+        {
+            "message": "Tx 12851857e1a929f490275e2850383484a3bed342d37a5db851748f888fe7ee01 failed. CreateAccount",
```

Comment:
I don't think that this is a good practice to show the whole stack trace. I think `Exception.getMessage()` is more than enough.

Reply:
ok